uber/site_sections/registration.py

```diff
@@ -71,6 +78,14 @@ def form(self, session, message='', return_to='', omit_badge='', **params):
             attendee.group_id = params['group_opt'] or None
             if AT_THE_CON and omit_badge:
                 attendee.badge_num = 0
+
```

Comment:
This will probably raise an exception if someone enters a number which is not an integer.

Also the third if-block doesn't have an ``int`` conversion, so it will probably always return true.